extensions/amp-analytics/0.1/vendors/blueconic.json

```diff
@@ -0,0 +1,10 @@
+{
+  "requests": {
+    "event": "https://${tenantId}/s/${pixelId}.gif?${trackingIdentifier}${defaultTrackingValues}${extraTrackingValues}"
+  },
+  "vars": {
+    "trackingIdentifier": "property=${property}&${property}=${clientId(cid-scope-cookie-fallback-name)}",
+    "defaultTrackingValues": "&sourceUrl=${sourceUrl}&canonicalUrl=${canonicalUrl}&referrer=${documentReferrer}&title=${title}&visitDate=${timestamp}&geoLocation=${ampGeo}&userAgent=${userAgent}",
```

Comment:
I would suggest you keep the values in `vars` as simple as possible.

Instead you could make trackingIdentifier a request:

```
"requests": {
    "event": "https://${tenantId}/s/${pixelId}.gif?${trackingIdentifier}...
    "trackingIdentifier": "property=${property}&${property}=${clientId(cid-scope-cookie-fallback-name)}".
}
```

This has the benefit of being a cleaner format and also not encoding the `=` in the requests (take a look at the failing travis test).

Reply:
Also move the defaultTrackingValues because the comment is also valid for those vars